pact/http/consumer-one/build.gradle.kts

```diff
@@ -15,3 +16,14 @@ dependencies {
     testImplementation("org.springframework.boot:spring-boot-starter-test")
     testImplementation("au.com.dius.pact.consumer:junit5")
 }
+
+project.version = "1.0.0"
+
+pact {
+    publish {
+        pactBrokerUrl = "http://localhost:9292"
+        consumerVersion = project.version
```

Comment:
Could this be set directly and the `project.version` left unset? Or are both needed for the plugin to work?

Reply:
They can be set independently. Using `project.version` just ensures they always align.